docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -139,10 +139,24 @@ function takes an input attestation key ID in its parameter list.
 With DCAP, the quote generation can be done either in-process,
 or out-of-process by working with a background service (called AESM) running
 on the same platform.
+  - On Linux platforms, access control for quote generation is enforced by
+  the SGX Linux driver.
+    - Every process that hosts the Quoting
+    Enclave (QE) is required to run in an account added to a special group `sgx_prv`,
+    as documented in the DCAP library [readme](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/README.md) and the DCAP driver [readme](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/driver/linux/README.md).
+      - Note: `sgx_prv` is an SGX provisioning access control mechanism implemented
+      in the DCAP driver since version V1.22, and in the new upstream SGX Linux driver.
+    - Impact to DCAP library usage:
+      - For in-process quote generation, every process that calls the DCAP library
+      is required to run in an account that belongs to the special `sgx_prv` group.
+      - On the other hand, for out-of-process quote generation, only the AESM
+      service process account needs to be added to the special group `sgx_prv`.
+  - Note: on Windows platforms quote generation access control stays the same.
+  QE's are required to be white-listed.
```

Comment:
Whitelisted how?  By what?  Can you provide the same level of detail for Windows as for Linux?

Reply:
Committed a new version with more details.